efs.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_efs_file_system" "efs_file_system" {
+  creation_token                  = var.creation_token_name
+  encrypted                       = false
+  performance_mode                = "generalPurpose"
+  throughput_mode                 = "bursting"
```

Comment:
provide functionality to change throughput_mode and also add provisioned_iops feature